README.md

```diff
@@ -62,3 +62,62 @@ import { useTranslation } from 'language/LanguageProvider';
+a website or an Android/iOS app through [Expo Go](https://expo.io/tools#client).
+
+### Building archive
+
+To build an Android or iOS archive run either `expo build:android` or `expo build:ios` in the project root.
+This will open a propt where you will be required to login to an expo account that will be responsible for the build.
+
+**Note:** If you're the publisher of the application remember that `expo build` will run `expo publish` automaticly if the build succeeds.
+
+When the build is done the archive will be available on your Expo account thourgh the link given by Expo CLI.
+For more build options please check out the Expo [documentation](https://docs.expo.io/distribution/building-standalone-apps/#3-start-the-build).
+
+
+## Translation
+
+Behaviour activation supports multiple luanguages and its easy to add additional languages.
+Adding a new language consists of two steps: 1. translations the actual app 2. tell the application that language exists.
+
+### Translating the application
+The first step in order to translate the application is to make a **copy** of the en english translation located at `src/language/en.tsx` and
+name it in accordance to the ISO 639-1 standard. (If you're unsure what language code to use you can find a list of them all over
+[here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)) You can now open the file and change the first occurance of `en` to your
+language code. You're now ready to start translating!
+
+Each line contains a so called key-value pair. The "key" (left) represent where in the application the text is located and the "value" (right)
+represents the text that is displayed. **Note:** The "keys" should always be the same and should not be changed.
+To translate the application simply change the "value" within the two " (quotation marks) to the translated word/sentence. If you're unsure how
+to translate a specific sentence check out the "key" for a clue of the context. It's also recommended to use the application itself to see where
+values are located.
+
+### Adding a new lanuage to the application
+To add a new language to the application we have to modify some of the code located in the `src/language/index.tsx`. If you're not a coder don't
+be alarmed! We will go through everything you need change step by step.
+
+1. Open the index file located at `src/language/index.tsx`.
+2. Copy the the following line `import en from 'language/en';` and replace `en` with your language code and place it bellow the other `import` lines.
+3. Add your language code to the line starting with `export type LanguageName = 'sv' ` by adding `| 'en'` replaced with your language code at the end of the line just before the `;` character. Example: `export type LanguageName = 'sv';` becomes `export type LanguageName = 'sv' | 'en;'.
+4. At last add `'en': en,` replaced with your language code just bellow the line `export const languages: Languages = {`.
```

Comment:
```suggestion
4. At last add `'en': en,` replaced with your language code just below the line `export const languages: Languages = {`. The further up you put it compared to other languages the further up it will appear in the language list in the app. Try to add your language so it is sorted alphabetically,
```

Reply:
I think this step could be hard to accomplish for someone with 0 programming skills and i think it's better for this to be resolved in a PR. The little things like "how far down can i placed the line?" could become problematic.